### PR TITLE
Rails 5.1.1 finished

### PR DESCRIPTION
**Pull request by bgeuken**

### CODE REVIEW

File: src/api/Gemfile [hunk 1]
```diff
@@ -61,7 +61,7 @@ gem 'feature'
 # for profiling
 gem 'peek'
 gem 'peek-dalli'
-gem 'peek-mysql2'
+gem 'peek-mysql2', git: 'https://github.com/kyohsuke/peek-mysql2.git'
```
Comment (hennevogel) [inline src/api/Gemfile]: This will not work. It won't be installed in the appliance...

File: .rubocop_todo.yml [hunk 2]
```diff
@@ -694,7 +694,6 @@ Style/PercentLiteralDelimiters:
     - 'src/api/app/controllers/source_controller.rb'
```
Comment (hennevogel) [inline .rubocop_todo.yml]: All of b11cea04e507d566298599603e82d78fdf2ac3f4 is already merged

Comment (bgeuken) [inline .rubocop_todo.yml]: Where?

Comment (hennevogel) [inline .rubocop_todo.yml]: ups. sorry

**Comment by bgeuken** [comment]
Tested the ui now. Everything seems to be working (and looking) normal.

**Comment by bgeuken** [comment]
Added the 'do not merge' label because it'S not clear right now, if our package builds can handle [fetching gems from github](https://github.com/openSUSE/open-build-service/pull/3261#discussion-diff-123227528R64)

**Comment by hennevogel** [comment]
@bgeuken what is not clear about this?

**Comment by bgeuken** [comment]
@hennevogel if our package script can handle this. and if we could fix this with reasonable effort.

Anyway, by now the new peek-mysql2 gem has been released 🎆

**Comment by bgeuken** [comment]
The test still fails with timeouts. I've added the capybara timeout fix from #3270

Let's see if that solves it.

**Pull request merged**